game/scripts/npc/abilities/chen_penitence.txt

```diff
@@ -42,12 +42,12 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "speed"                                           "2000 2000 2000 2000 2000 2000"
+        "speed"                                           "2000"
       }
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_movement_speed"                            "-18 -24 -30 -36 -42 -50"
+        "bonus_movement_speed"                            "-18 -24 -30 -36 -42 -48"
```

Comment:
because movement speed and bonus damage taken have equal values at their levels. This doesn't seem like a difference made out of balance reasons but just it being a typo